js/a11y/browserFocus.js

```diff
@@ -45,19 +45,31 @@ export default class BrowserFocus extends Backbone.Controller {
     if (event.target !== event.currentTarget) {
       return;
     }
-    // Do not auto next if the focus isn't returning to the body
-    if (!$(event.relatedTarget).is('body')) {
+    // Do not auto next if the focus isn't returning to the body or html element
+    if (!$(event.relatedTarget).is('body, html')) {
       return;
     }
```

Comment:
It looks like the same issue occurs after submitting a question and then clicking on one of the options again. At that point these conditions are met. Is it acceptable to move these conditions to execute after https://github.com/adaptlearning/adapt-contrib-core/pull/697/files#diff-3ff4169174c1fed2be880d3992f0884ac57b087ee5195aad74c799717f73a8b8R55-R61 rather than before?